### PR TITLE
API mock server

### PR DESCRIPTION
**Pull request by naganoyasuaki**
Added examples of 5.using-with-api-mock

https://user-images.githubusercontent.com/1168878/168738818-54f3f4a8-2cd8-45f4-a806-ff6af8511303.mp4

### CODE REVIEW

File: examples/5.using-with-api-mock/index.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        transition: all 0.3s ease-in-out;
+      }
+
+      #chat-button:hover {
+        cursor: pointer;
+        background-color: rgba(54, 129, 254, 0.8);
+        box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.5);
+      }
+    </style>
+    <script src="https://naganoyasuaki.github.io/clova-chatbot-web-kit/examples/1.getting-started/webchat-3.js"></script>
```
Comment (naganoyasuaki) [inline examples/5.using-with-api-mock/index.html]: Should be replaced by CDN

Comment (Ryo0o) [reply examples/5.using-with-api-mock/index.html]: Does this not work with webchat.js currently deployed on the CDN?

Comment (naganoyasuaki) [reply examples/5.using-with-api-mock/index.html]: @Ryo0o It works, but
https://github.com/line/clova-chatbot-web-kit/pull/7
is not included, so `ONLY NORMAL IMAGE` response is not displayed.

Comment (naganoyasuaki) [reply examples/5.using-with-api-mock/index.html]: スクリーンショット 2022-05-24 11 26 39

Comment (Ryo0o) [reply examples/5.using-with-api-mock/index.html]: I see.
I will release a new version of webchat.js and put it on the CDN.
Please wait a moment.

Comment (Ryo0o) [reply examples/5.using-with-api-mock/index.html]: I deployed new webchat.js as `webchat-1.2.0.js`
Please check it, and update this PR.

**Review by Ryo0o — CHANGES_REQUESTED** [review]
I deployed new webchat.js as [webchat-1.2.0.js](https://clovachatbot-webkit.line-scdn.net/webchat/webchat-1.2.0.js)
Please check it, and update this PR.

**Review by Ryo0o — APPROVED** [review]
LGTM

**Pull request merged**